src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Sdk/Sdk.props

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
+    <UseMonoRuntime Condition="'$(RuntimeIdentifier)' == 'Browser-wasm'">true</UseMonoRuntime>
```

Comment:
I think/hope this is no longer needed after my #55258

Reply:
Your PR changes the InTree, and LocalBuild targets, which don't get used for the workloads.

Reply:
Follow up PR? :) And if we are moving it to wasm sdk, then might as well move it to wasm props, then it won't be needed in InTree/LocalBuild either.